src/osip-monitoring-view/src/main/java/edu/kit/pse/osip/monitoring/view/settings/SettingsMainWindow.java

```diff
@@ -120,6 +120,11 @@ private void createLayout() {
         Scene scene = new Scene(generalLayout);
         window.setTitle(translator.getString("monitoring.settings.title"));
         window.getIcons().add(new Image("icon.png"));
+        window.setOnCloseRequest(event -> {
+            if (buttonCancel.isDisabled()) {
+                event.consume();
+            } 
+        });
```

Comment:
Hmm ja das wäre in Ordnung. Also wenn man auf das X drückt kommt ein Popup der einen darauf hinweist, dass man die Einstellungen richtig setzen soll. Den Cancel Button würde ich trotzdem deaktivieren, oder?